cmd/authorize/authorize.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/rclone/rclone/fs/config"
 	"github.com/rclone/rclone/fs/config/flags"
 	"github.com/spf13/cobra"
+	"github.com/rclone/rclone/lib/oauthutil"
```

Comment:
The lint error is because the imports should be in alphabetical order, I'm using VSCode with go extension which does this automatically on save, but I think you can use the go fmt tool to do it file by file as well:

`gofmt -w cmd/authorize/authorize.go`

Reply:
I ran `go fmt` from the root of the repository, I'm not sure why it didn't catch it. I'll get that fixed though.